acquire/outputs/tar.py

```diff
@@ -1,12 +1,28 @@
 import io
 import tarfile
+from pathlib import Path
+from typing import BinaryIO, Optional
+
+from dissect.target.exceptions import FileNotFoundError as TargeFileNotFoundError
```

Comment:
```suggestion
from dissect.target.exceptions import FileNotFoundError as DissectFileNotFoundError
```
`TargetFileNotFound` is technically correct but makes me think too much of a "target file", as in destination.

Reply:
Changed this to just `from dissect.target.exceptions import FileNotFoundError`